orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/InSet.feature

```diff
@@ -996,24 +996,14 @@ Feature: User can specify that a field value belongs to a set of predetermined o
 
 ### Financial data types ###
 
-  Scenario: In set of valid ISINs combined with an ISIN constraint returns members of the set
-    Given there is a field foo
-    And foo is in set:
-      | "GB00YG2XYC52" |
-    And foo is of type "ISIN"
-    Then the following data should be generated:
-      | foo            |
-      | null           |
-      | "GB00YG2XYC52" |
-
```

Comment:
Why has this test been deleted?

Reply:
duplicate of tests in ofTypeFinancialCodes.feature